doc/classes/Plane.xml

```diff
@@ -170,10 +170,13 @@
 	</members>
 	<constants>
 		<constant name="PLANE_YZ" value="Plane( 1, 0, 0, 0 )">
+			A plane that extends in the Y-axis and Z-axis.
 		</constant>
 		<constant name="PLANE_XZ" value="Plane( 0, 1, 0, 0 )">
+			A plane that extends in the X=axis and Z-axis.
```

Comment:
You use '=' here. IMO it could be "X and Z axes" without hyphen. Same for other two descriptions, and third one has an extra 'the'.